### PR TITLE
issue #72 - modified the main head and contact button

### PR DESCRIPTION
**Pull request by pbpcodes**
1st PR for Algo-Phantoms-Frontend through GSSoC'21.
Modified the following things: 

1) Main head text positioning
2) Corrected the typo in "focusses" to "focusing"
3) Added functionality to the Contact Us button with some styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,7 +73,7 @@
 
         <div class="banner-text text-center">
             <p class="main-head">We are <span>Programmers</span></p>
-            <p> Algo Phantoms is an Open Source organization focusses on Data Structure and Algorithms</p>
+            <p class="below-head"> Algo Phantoms is an <span >Open Source organization</span> focusing on <span>  Data Structure and Algorithms</span></p>
```
Comment (harikesh409) [inline index.html:76]: It is not an typo as everywhere it is present as focusses only.

File: index.html [hunk 2]
```diff
@@ -133,7 +133,7 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
             <div class="row">
                 <div class="col-md-3 col-sm-6 col-lg-3 col-12">
                     <div class="">
-                        <img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100px" height="100px">
+                        <a href="index.html"><img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100px" height="100px"> </a>
```
Comment (harikesh409) [inline index.html:136]: Link is not needed over here.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Resolve merge conflicts and do the mentioned changes.

File: index.html [hunk 3]
```diff
@@ -155,7 +155,7 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
             <div class="row">
                 <div class="col-md-3 col-sm-6 col-lg-3 col-12">
                     <div class="">
-                        <a href="index.html"><img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100px" height="100px"> </a>
+                        <a href=""><img src="./images/icon.png" alt="" class="img-fluid mb-lg-4" width="100px" height="100px"> </a>
```
Comment (harikesh409) [inline index.html:158]: No need of an anchor tag, with blank anchor tag like this if we click on it the page will reload which is unnecessary.

File: index.html [hunk 4]
```diff
@@ -140,11 +140,11 @@ <h2 class="text-white font-weight-bold py-2">Contact Us</h2>
                 <div class="col-lg-6 col-md-6 col-sm-12 col-12">
                     <form action="">
                         <div class="input-group">
-                            <input type="text" class="form-control py-3" placeholder="Enter your Email">
+                            <input type="email" class="form-control py-3" placeholder="Enter your Email">
                             <div class="input-group-append">
-                                <div class="input-group-text subscribe-button text-white px-5 font-weight-bold">
+                                <button type="submit" class="input-group-text subscribe-button text-white px-5 font-weight-bold">
```
Comment (harikesh409) [inline index.html:145]: No need of button type submit as there is no form action as of now.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (pbpcodes) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (pbpcodes) [reply index.html]: done

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**